README.md

```diff
@@ -1,5 +1,5 @@
-# DSharpPlusNextGen ![GitHub](https://img.shields.io/github/license/Aiko-IT-Systems/DSharpPlusNextGen?label=License) ![Sponsors](https://img.shields.io/github/sponsors/Lulalaby?label=Sponsors)  ![Discord Server](https://img.shields.io/discord/858089281214087179.svg?label=Discord)
-Discord Bot Library written in C# for .NET - based off [DSharpPlus](https://github.com/DSharpPlus/DSharpPlus).
+# DisCatSharp ![GitHub](https://img.shields.io/github/license/Aiko-IT-Systems/DSharpPlusNextGen?label=License) ![Sponsors](https://img.shields.io/github/sponsors/Lulalaby?label=Sponsors)  ![Discord Server](https://img.shields.io/discord/858089281214087179.svg?label=Discord)
+Discord Bot Library written in C# for .NET.
```

Comment:
I would add something like "originally based off DSharpPlus"

Reply:
yes

Reply:
@OoLunar what do u think

Reply:
I think the "Written in C# for .NET" part is a bit redundant, considering .NET is an ecosystem that I believe Microsoft is trying to turn into one general thing, if not phase out entirely. Perhaps something like "Originally based off of DSharpPlus, DisCatSharps aims to fix DSP's internal mistakes, without any internal drama."

Reply:
yeah seems right